<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoControlBar</class>
 <widget class="QWidget" name="VideoControlBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>73</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoControlBar</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">VideoControlBar{background-color: rgba(0, 0, 0 , 0.3);}</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>891</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
    <property name="spacing">
     <number>0</number>
    </property>
    <item>
     <widget class="QSlider" name="horizontalSlider">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider {
    /* 整体轨道样式 */
    background: transparent;
    height: 4px;  /* 水平滑块的高度 / 垂直滑块的宽度 */
}

/* 已过进度部分（蓝色） */
QSlider::sub-page:horizontal {
    background: #1E90FF;  /* 蓝色进度 */
}

QSlider::add-page:horizontal {
    background: #CCCCCC;  /* 未完成部分灰色 */
}

/* 滑块（默认隐藏，悬停时显示） */
QSlider::handle:horizontal {
    background: white;
    width: 12px;    /* 滑块宽度 */
    height: 12px;   /* 滑块高度 */
    margin: -4px 0;  /* 使滑块超出轨道 */
    border-radius: 6px;
    border: 1px solid #999;
    opacity: 0;      /* 默认透明 */
}

/* 鼠标悬停时显示滑块 */
QSlider::handle:horizontal:hover {
    opacity: 1;      /* 完全显示 */
}</string>
      </property>
      <property name="maximum">
       <number>35</number>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="start">
        <property name="text">
         <string>暂停/开始</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="cur_label">
        <property name="styleSheet">
         <string notr="true">color:rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string>00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="duration_label">
        <property name="styleSheet">
         <string notr="true">color:rgb(255,255,255)</string>
        </property>
        <property name="text">
         <string>/00:00</string>
        </property>
        <property name="textFormat">
         <enum>Qt::TextFormat::AutoText</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <property name="font">
         <font>
          <stylestrategy>PreferAntialias</stylestrategy>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox::drop-down { border: 0px; }QComboBox::down-arrow { width: 0px; height: 0px; } </string>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <property name="modelColumn">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>x0.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x1.0</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x1.5</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>x2.0</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sound_button">
        <property name="text">
         <string>声音</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="maximize_button">
        <property name="text">
         <string>放大/缩小</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Policy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
